### PR TITLE
Generalcomponents

### LINKED ISSUE #12: As a developer, I need buttons and inputs so I can use them for many different pages.

### PR DESCRIPTION
**Pull request by franzosterkamp**
closes #12

### CODE REVIEW

File: client/src/stories/Button.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from '../components/General/Button';
```
Comment (manuelwecker) [inline client/src/stories/Button.stories.js]: General as folder name in Capital letters?

File: client/src/components/General/TextInput.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled';
+
+const TextInput = styled.textarea`
+  font-weight: bold;
```
Comment (holgermanzke) [inline client/src/components/General/TextInput.js]: Did you tried not to use font-weight: bold? I think that a bold font for small letters looks not so pretty.

Comment (franzosterkamp) [reply client/src/components/General/TextInput.js]: thank for the advice, but i think i will keep it for the moment like this.

File: client/src/stories/Button.stories.js (hunk 1 above)
Comment (franzosterkamp) [reply client/src/stories/Button.stories.js]: Good advice, i did it in all folders wrong :D

**Pull request merged**